linode/linode_client.py

```diff
@@ -283,6 +283,22 @@ def assign_ips(self, region, *assignments):
             ips.append(i)
 
         return ips
+    
+    def add_ip(self, linode_id):
```

Comment:
In the Linode object, we call this [allocating](https://github.com/linode/python-linode-api/blob/master/linode/objects/linode/linode.py#L296)